File: src/main/java/se/kth/spork/exception/MergeException.java
package se.kth.spork.exception;

/**
 * Generic exception thrown when there is a critical error in the merge.
 *
 * @author Simon Larsén
 */
public class MergeException extends SporkException {
    public MergeException(String s) {
        super(s);
    }

    public MergeException(String s, Throwable throwable) {
        super(s, throwable);
    }
}

File: src/main/kotlin/se/kth/spork/exception/MergeException.kt
package se.kth.spork.exception

/**
 * Generic exception thrown when there is a critical error in the merge.
 *
 * @author Simon Larsén
 */
class MergeException @JvmOverloads constructor(s: String, t: Throwable? = null) : SporkException(s, t)
